contrib/docker_packer/main.go

```diff
@@ -0,0 +1,73 @@
+package main
+
+import (
+	"context"
+	"encoding/base64"
+	"encoding/json"
+	"fmt"
+	"log"
+	"os"
+
+	runtime "github.com/metrue/fx/container_runtimes/docker/sdk"
+	"github.com/metrue/fx/packer"
+)
+
+// Version binary version
+var Version = "0.0.1"
+
+func init() {
+	os.Setenv("DEBUG", "true")
```

Comment:
I would like to check the log for the current stage, will clean up when `fx` stable.